hist/hist/src/TF1.cxx

```diff
@@ -605,11 +606,11 @@ TF1::TF1(const char *name, const char *formula, Double_t xmin, Double_t xmax, EA
       }
 
       // Then check if we need NSUM syntax:
-   } else if (TString(formula, 5) == "NSUM(" && formula[strlen(formula) - 1] == ')') {
+   } else if (TString(formula, 5) == "NSUM(" && formula[formulaLength - 1] == ')') {
```

Comment:
This is the same error as above. It's probably good to also use `strncmp` here.

Reply:
Ah okay. As I understand, strncmp compares two strings until the two differ of until a null-character is used to terminate them. How exactly could this be used here to check if formula ends with ")"?

Reply:
```suggestion
   } else if (strncmp(formula, "NSUM(", 5) == 0 && formula[formulaLength - 1] == ')') {
```